Makefile

```diff
@@ -119,18 +119,11 @@ targets:	## Display available targets and descriptions
 
 ### Versioning Targets ###
 
-# Bumpversion 'dry' config
-# if 'dry' is specified as target, any bumpversion call using 'BUMP_XARGS' will not apply changes
-BUMP_XARGS ?= --verbose --allow-dirty	## Extra arguments to pass down to version control utility
-BUMP_XARGS := $(call clean_opt,$(BUMP_XARGS))
-ifeq ($(filter dry, $(MAKECMDGOALS)), dry)
-  BUMP_XARGS := $(BUMP_XARGS) --dry-run
-endif
-BUMP_CFG  ?= .bumpversion.cfg				## Bump version configuration (default recommended)
+BUMP_CFG  ?= .bumpversion.toml				## Bump version configuration (default recommended)
 BUMP_CFG  := $(call clean_opt,$(BUMP_CFG))
-BUMP_TOOL := bump2version
+BUMP_TOOL := bump-my-version
 BUMP_PATH := $(CONDA_ENV_PATH)/bin/$(BUMP_TOOL)
-BUMP_CMD  := $(BUMP_TOOL) --config-file "$(BUMP_CFG)"
+BUMP_CMD  := $(BUMP_TOOL)
```

Comment:
Leave the `BUMP_CFG` reference in `BUMP_CMD` as it was. This allows using different configurations to tests alternate definitions. Right now, this change makes the option useless as it won't take effect.

Reply:
This has been reverted.